dask/dataframe/core.py

```diff
@@ -5619,6 +5645,7 @@ def map_partitions(
     meta=no_default,
     enforce_metadata=True,
     transform_divisions=True,
+    length_preserving=False,
```

Comment:
Nitpick, but the other kwargs are `{verb}_{noun}` how would you feel about `preserve_length`

Reply:
Makes sense. I have no idea what to call this - So, your opinion is useful, and not just a nitpick :)